server/plugin/graphql/lhs_request.go

```diff
@@ -29,64 +29,66 @@ func (c *Client) GetLHSData(ctx context.Context) ([]*github.Issue, []*github.Iss
 		queryParamOpenPRsCursor:     (*githubv4.String)(nil),
 	}
 
-	if c.org != "" {
-		params[queryParamOpenPRQueryArg] = githubv4.String(fmt.Sprintf("org:%s %s", c.org, params[queryParamOpenPRQueryArg]))
-		params[queryParamReviewPRQueryArg] = githubv4.String(fmt.Sprintf("org:%s %s", c.org, params[queryParamReviewPRQueryArg]))
-		params[queryParamAssigneeQueryArg] = githubv4.String(fmt.Sprintf("org:%s %s", c.org, params[queryParamAssigneeQueryArg]))
-	}
+	orgsList := c.getOrganizations()
 
 	var resultReview, resultAssignee, resultOpenPR []*github.Issue
-	allReviewRequestsFetched, allAssignmentsFetched, allOpenPRsFetched := false, false, false
-
-	for {
-		if allReviewRequestsFetched && allAssignmentsFetched && allOpenPRsFetched {
-			break
-		}
+	for _, org := range orgsList {
+		params[queryParamOpenPRQueryArg] = githubv4.String(fmt.Sprintf("org:%s %s", org, params[queryParamOpenPRQueryArg]))
+		params[queryParamReviewPRQueryArg] = githubv4.String(fmt.Sprintf("org:%s %s", org, params[queryParamReviewPRQueryArg]))
+		params[queryParamAssigneeQueryArg] = githubv4.String(fmt.Sprintf("org:%s %s", org, params[queryParamAssigneeQueryArg]))
```

Comment:
Any way we can "or" these together in a GitHub query? Also it seems that if the GitHub org is not set in the config, then we end up not fetching anything right? Because we won't enter into this loop

Reply:
I understand now, these are appending params to the requests that will be sent. Confirmed that tacking on multiple `org:%s` clauses to the same GitHub query does what we want it to do